src/reducers/index.test.ts

```diff
@@ -37,4 +49,23 @@ it('setGamesList', () => {
   expect(reduce(initialState, action)).toEqual(expectedState);
 });
 
+it('setSignals', () => {
+  let signalsList = [sigEntry];
+  let action: Action = {
+    setSignals: signalsList
+  };
+  let initialState = storeStateDefault;
+  let expectedState = Object.assign(storeStateDefault, {signalsList: signalsList});
```

Comment:
For the things which are not going to change, you should use `const` instead of `let`. Everything here isn't going to change.